core/templates/pages/topics-and-skills-dashboard-page/topics-list/topics-list.directive.html

```diff
@@ -191,8 +191,8 @@
     margin: 0;
   }
   topics-list .topic-edit-options {
-    background-color: #ffffff;
-    border: 1px solid black;
+    background-color: #fff;
```

Comment:
Use 6 char hex values here and below.

Reply:
I am not very sure about this. Sandeep told me to use 3 char hex value if the values are same. You can look at this [here](
https://github.com/oppia/oppia/pull/9719#discussion_r447109600).